pkg/cmd/kafka/describe/describe.go

```diff
@@ -125,7 +127,11 @@ func runDescribe(opts *Options) error {
 		}
 	}
 
-	return printKafka(kafkaInstance, opts)
+	if !opts.bootstrapServer {
+		return printKafka(kafkaInstance, opts)
+	}
+	opts.Logger.Info(*kafkaInstance.BootstrapServerHost)
```

Comment:
It needs to print to stdout.

The logger only prints to stderr, which is not machine-readable and so is not useful for this.